specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -6835,21 +6835,12 @@
           "type": "object",
           "properties": {
             "computeType": {
-              "description": "Compute type of the cluster which will execute data flow job.",
-              "type": "string",
-              "enum": [
```

Comment:
This would cause breaking change in SDK. Please request an approval from API Review Board as defined in the Breaking Change Policy.

Reply:
@qianwens This is actually a bug that we are trying to fix. I agree that customers may have to update their code if they are relying on the string data type for these two properties, and the change required would be a simple cast to string type. But this is still a bug and we are not able to support expressions for these two properties as they are currently modeled as string. Also, we do have some customers waiting on the expression support. Given this, do we still need to get approval from API Review Board?